Cargo.toml

```diff
@@ -13,10 +13,11 @@ categories = ["concurrency", "memory-management"]
 
 [features]
 nightly = []
-strict_gc = []
+sanitize = []
```

Comment:
Do we not want two orthogonal `sanitize` and `strict_gc` features?

Reply:
I think the strictness should be configurable with more flexibility: the tradeoff between latency and peak working set is one of the most important parameters for garbage collection.  But I think we can tentatively remove `strict_gc` support now.  Anyway, the size of a bag seems too coarse-grained a parameter for the strictness.